conf/modules.config

```diff
@@ -822,6 +822,39 @@ process {
         ext.prefix = { "${meta.assembler}-COMEBin-${meta.id}" }
     }
 
+    withName: METABINNER {
+        publishDir = [
+            [
+                path: { "${params.outdir}/GenomeBinning/MetaBinner/stats"},
+                mode: params.publish_dir_mode,
+                pattern: '*.{log,log.gz,tsv.gz}'
+            ],
+            [
+                path: { "${params.outdir}/GenomeBinning/MetaBinner/"},
+                mode: params.publish_dir_mode,
+                pattern: 'bins/*.fa.gz'
+            ],
+            [
+                path: { "${params.outdir}/GenomeBinning/MetaBinner/discarded" },
+                mode: params.publish_dir_mode,
+                pattern: '*tooShort.fa.gz'
+            ],
+            [
+                path: { "${params.outdir}/GenomeBinning/MetaBinner/discarded" },
+                mode: params.publish_dir_mode,
+                pattern: '*lowDepth.fa.gz'
+            ],
+            [
+                path: { "${params.outdir}/GenomeBinning/MetaBinner/unbinned" },
+                mode: params.publish_dir_mode,
+                pattern: '*unbinned.fa.gz'
+            ]
+        ]
+        ext.prefix = { "${meta.assembler}-MetaBinner-${meta.id}" }
+        ext.min_contig_size = { params.min_contig_size < 1000 ? "1000" : "${params.min_contig_size}" }
+        ext.args = "-s large"
```

Comment:
I think this should be a configurable user-facing option rather than hardcoding:

```
        -s STR          Dataset scale; eg. small,large,huge (default:large); Users can choose "huge" to run MetaBinner on huge datasets
                        with lower memory requirements.
```

Reply:
Well, atm its configurable with a config. I have too less experience with that tool to argue whether that will be a commonly adjusted param or not, I can expose it though, why not.

Reply:
done I think